OpenRA.Mods.Common/Widgets/Logic/Ingame/IngameMenuLogic.cs

```diff
@@ -351,12 +351,26 @@ void CreateSaveMapButton()
 				hideMenu = true;
 				var editorActorLayer = world.WorldActor.Trait<EditorActorLayer>();
 				var actionManager = world.WorldActor.Trait<EditorActionManager>();
+
+				var playerDefinitions = editorActorLayer.Players.ToMiniYaml();
+
+				var playerCount = new MapPlayers(playerDefinitions).Players.Count;
+				if (playerCount > 64)
+				{
+					ConfirmationDialogs.ButtonPrompt(
+					title: "Error: Max player count exceeded",
+					text: $"There are too many players defined ({playerCount}/64).",
```

Comment:
We so far haven't used any interpolated strings in the code, so can we stick with `"There are too many players defined ({0}/64).".F(playerCount)` until we make a decision (and ideally a porting effort) to migrate properly?

Reply:
Of course! Changed to your proposed code.